docs/usage/direct_api.md

```diff
@@ -0,0 +1,56 @@
+---
+jupytext:
+  cell_metadata_filter: -all
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.7.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Direct Widget API
+
+While the primary API is to use the {func}`@magicgui <magicgui.magicgui>`
+decorator, one can also instantiate widgets directly using any of the widgets in
+{mod}`magicgui.widgets`.
+
+Generally, one will construct a {class}`~magicgui.widgets.Container` object that
+acts as a layout for all of the sub-widgets.
+
+```{code-cell} python
+from magicgui.widgets import *
```

Comment:
is this really necessary @tlambert03? 😂

Reply:
hah.  almost certainly never.  I don't know what compelled me. Felt like the kinda thing people do in this scenario?  will undo, don't want to get people using that.